src/main/kotlin/uk/gov/dluhc/printapi/messaging/ProcessPrintResponseMessageListener.kt

```diff
@@ -24,7 +24,10 @@ class ProcessPrintResponseMessageListener(
     override fun handleMessage(@Payload payload: ProcessPrintResponseMessage) {
         logger.info { "Begin processing PrintResponse with requestId ${payload.requestId}" }
         printResponseProcessingService.processPrintResponse(payload)?.also {
-            statisticsUpdateService.triggerVoterCardStatisticsUpdate(it.sourceReference!!)
+            if (payload.isFromApplicationsApi == true)
```

Comment:
We're reusing this `if(payload.isFromApplicationsApi)` logic in a bunch of places, can we instead have this handled in the statistics update service to avoid replicating code